src/main/resources/assets/goodgenerator/data/essentia.json

```diff
@@ -304,13 +304,13 @@
       "name": "Strontio",
       "fuelValue": 200,
       "category": "SPRITE",
-      "consumeCeo": 0.03
+      "consumeCeo": 0.02
```

Comment:
What's the exact formula here?

Reply:
The value of consumeCeo represents the fluid consumption rate multiplier when this essentia is used. For example, if there is an essentia with a consumptionCeo of 10, and it is used to consume fluid at a base rate of 1000 mB/t, then the consumptionCeo will take the fluid consumption rate to 10,000 mB/t. The larger the value, the higher the cost of generating electricity

Reply:
consumeCeo has nothing to do with final power generation, which is penalized by the use of powerful essentia